figures/helpers.py

```diff
@@ -241,3 +251,124 @@ def get_required_registration_fields_for_user(user, site):
         if hasattr(user.profile, field)
     ]
     return user_required_fields
+
+
+def _get_completed_courses(courses):
+    """
+    Return number of completed courses from list of courses.
+    """
+    courses_count = 0
+    for course in courses:
+        if course['progress_data'] and course['progress_data']['course_completed']:
+            courses_count += 1
+
+    return str(courses_count)
+
+
+def send_email_with_attachment(recipient_emails, subject=PDF_EMAIL_SUBJECT, body=PDF_EMAIL_BODY, pdf_file=None):
+    """
+    Send email with attachments to given recipients.
+    """
+    message = EmailMessage()
+    message.to = recipient_emails
+    message.subject = subject
+    message.body = body
+    if pdf_file:
+        message.attach('{}.pdf'.format(LEARNERS_OVERVIEW_REPORT), pdf_file, 'application/pdf')
+
+    try:
+        message.send(fail_silently=False)
```

Comment:
@taimoor-ahmed-1 can we use `edx-ace` https://github.com/edx/edx-ace ?

Reply:
`edx-ace` wasn't able to handle attached files, so we have Django Email for html based email.